python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py

```diff
@@ -0,0 +1,226 @@
+        )
+
+        # Average time for a bump test is 23s
+        self.timeout_one_bump = 25
+
+        # Getting ids from mtcs
+        self.m1m3_actuator_ids = self.mtcs.get_m1m3_actuator_ids()
+        self.m1m3_secondary_actuator_ids = self.mtcs.get_m1m3_actuator_secondary_ids()
+
+        self.total_actuators = len(self.m1m3_actuator_ids)
+        self.total_sec_actuators = len(self.m1m3_secondary_actuator_ids)
+
+        # Getting actuators ids
+        self.actuators = self.m1m3_actuator_ids
+
+    async def assert_feasibility(self):
+        """Verify that the system is in a feasible state before
+        running bump test.
+        """
+
+        for comp in self.mtcs.components_attr:
+            if comp != "mtm1m3":
+                self.log.debug(f"Ignoring component {comp}.")
+                setattr(self.mtcs.check, comp, False)
+
+        await asyncio.gather(
+            self.mtcs.assert_all_enabled(),
+            self.mtcs.assert_liveliness(),
+        )
+
+    @classmethod
+    def get_schema(self, cls):
+
+        m1m3_actuator_ids_str = ', '.join(str(fa) for fa in self.m1m3_actuator_ids)
+
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/m1m3/check_actuators.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: CheckAcutators v1
+        description: Configuration for Maintel bump test SAL Script.
+        type: object
+        properties:
+            actuators:
+                description: Actuators to run the bump test.
+                oneOf:
+                  - type: array
+                    items:
+                      type: number
+                      enum: [{m1m3_actuator_ids_str}]
+                    minItems: 1
+                    uniqueItems: true
+                    additionalItems: false
+                  - type: string
+                    enum: ["all"]
+                default: "all"
+        additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+   
+    async def configure(self, config):
+        """Configure the script.
+        
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration
+        """
+
+        # Actuators defined as "all" (ID) or a list of actuators (INDEX)
+        self.actuators = (
+            self.m1m3_actuator_ids if config.actuators == "all" else config.actuators
+        )
+
+        # Check if actuators are valid
+        if not all([actuator in self.m1m3_actuator_table_id for actuator in self.actuators]):
+            raise ValueError("Invalid actuator ID.")
+
+    def set_metadata(self, metadata):
+        
+        metadata.duration = self.timeout_one_bump * (self.total_actuators + self.total_sec_actuators) 
+
+    def is_actuator_dual(self, actuator_id: int) -> bool:
```

Comment:
Can you rename this method? Consider something like `is_actuator_secondary` or `has_secondary_actuator`. That is the nomenclature in the m1m3 interface.